mental-model - the stairs (#15)
guillaumewuip/guillaumewuip.github.io: _mentalModels/2022-03-20-stairs.md

---
title: The Stairs

draft: false

page : false 

image: /assets/img/mental-models/stairs.svg

excerpt: >
  The Stairs is a good metaphor to think about _progression_- specially with a human factor.

---

<img
  alt="A schematic stairs pattern"
  src="/assets/img/mental-models/stairs.svg"
  style="max-height: 480px; margin: 2rem auto"
/>

The Stairs is a good metaphor to think about _progression_- specially with a
human factor.

> It could be about how a team project is going at work. Or the typical learning
> curve of mastering a new language. Or a personal advancement toward a career
> goal. This is never linear.

Reviewer: ```suggestion
It could be about the way a team project is going at work, or the typical learning
curve of mastering a new language. It can even be about one's personal advancement toward a career
goal. It is somewhat linear when zoomed out (there's a starting line and a goal), but if you zoom in, you'll find that it's not a straight line.
```

---
title: The Stairs

draft: false

page : false 

image: /assets/img/mental-models/stairs.svg

excerpt: >
  The Stairs is a good metaphor to think about _progression_- specially with a human factor.

---

<img
  alt="A schematic stairs pattern"
  src="/assets/img/mental-models/stairs.svg"
  style="max-height: 480px; margin: 2rem auto"
/>

The Stairs is a good metaphor to think about _progression_ - specially with a
human factor.

It could be about the way a team project is going at work, or the typical
learning curve of mastering a new language. It can even be about one's personal
advancement toward a career goal.

It could looks like linear at first thought. We have a starting line and a goal.
Isn't the progression like the straight line between the two? It's certainly
not. Taking a step back for the progression itself helps us understand the real

> pattern.

Reviewer: ```suggestion
It looks linear at first. We have a starting line and a goal.
Isn't progression like a straight line between the two? Certainly
not! Progression not being linear is the core of the pattern.
```

> This is never linear. We alternates between moments of slow and quick
> progression. It's like going up a stair with very long step.

Reviewer: ```suggestion
We alternate between moments of slow and quick
progression. It's like going up a stair where each step has a very long thread and a tall riser at the end.
```

https://www.stair-parts.com/wp-content/uploads/staircase-diagram-parts-of-a-staircase-stair-parts-com.jpg

J'arrive pas à croire que je met cette image dans une peer review.

<img
  alt="A schematic stairs with two tiny persons. One is walking. One is climbing a step"
  src="/assets/img/mental-models/stairs-people.svg"
  style="max-height: 480px; margin: 2rem auto"
/>

> There are moments where we're walking peacefully on one step. We're not making
> much visible progress but we're certainly progressing towards the next step.
> We're consolidating previous advancements, accumulating experience at a
> sustainable pace.

Reviewer: ```suggestion
There are moments where we're walking peacefully on one step. We're not making
much visible progress but we're still progressing towards the next step.
We're consolidating previous advancements, accumulating experience at a
sustainable pace.
```

> There are moments where we surprise ourself climbing a step. We have the
> sensation of having made deep progress in a very short time. A fundamental
> concept just clicked. The team delivered a important component of the project.
> Personal and company goals finally compound.

Reviewer: ```suggestion
Then eventually come moments where we find ourselves climbing up to the next step. We then feel like we've made deep progress in a very short time. A fundamental
concept just clicked. The team delivered an important component for the project.
Personal and company goals finally compound. Something noticeable has been achieved.
```
>  in a very short time

T'es sûr ? La métaphore peut aussi faire passer cette marche comme un mur à escalader, ce qui peut être long.

Author: arg ouais, je vais mettre une ligne du temps peut-être

> We alternate between these two very different kinds of progression moments.

Reviewer: ```suggestion
Progression is a back-and-forth between these two very different moments.
```

---

> Asking ourself where we are on The Stairs is important. Are we "climbing a
> step"? Then dedicating a bit more energy into it could help giving it a helpful
> push.

Reviewer: ```suggestion
Asking ourselves where we currently are on The Stairs is important. Are we "climbing a
step"? Then dedicating a bit more energy into it could help us reach a new and valuable higher floor.
```

> Are we on walking on a long step? Then let's not push for it to much. We're on a
> marathon, not a sprint, and we should keep energy for climbing the next step.

Reviewer: ```suggestion
Are we walking on the long thread of a step? Then let's not push for it too much. We're on a
marathon, not a sprint. We should keep our energy for when the time comes to climb the riser to the next step.
```
